provider-gui/components/AccountPicker.tsx

```diff
@@ -0,0 +1,68 @@
+'use client'
+
+import { FormControl, FormGroup, MenuItem, Select } from '@mui/material'
+import { InjectedAccountWithMeta } from '@polkadot/extension-inject/types'
+import { SelectChangeEvent } from '@mui/material/Select'
+import { networks } from '@/types/GlobalStateTypes'
+import { useEffect, useState } from 'react'
+import { useGlobalState } from '../contexts/GlobalContext'
+import { web3Accounts, web3Enable } from '@polkadot/extension-dapp'
+import React from 'react'
+
+interface Injected {
+    enable: typeof web3Enable
+    accounts: typeof web3Accounts
+}
+
+const injected: Injected = {
```

Comment:
is this needed? can't you access web3Enable and web3Accounts directly without this wrapper?

Reply:
Done